pawan/html/html_assignment3.txt

```diff
@@ -0,0 +1,15 @@
+Q 3.1 Specify two advantages of adding the ‘lang’ attribute to a web page
+	1.The lang attribute takes an ISO language code as its value. Typically this is a two letter code such as “en” for English, but it can also be an extended code such as “en-gb” for British English.
```

Comment:
do you know what is ISO language ?

Reply:
no sir i got this answer from google

Reply:
I knew that, thats why I asked you that question about ISO.
You need to first understand the question, then answer it. Anybody can just search the google, but dont be that anybody. you should be on your own.
write answer in your words, after understanding the question.